test/test_client_c6u.py

```diff
@@ -10,10 +10,50 @@
     Device,
     ClientException,
     VPN,
+    GuestWifiStatus,
 )
 
 
 class TestTPLinkClient(TestCase):
+    def test_get_guest_wifi_info(self) -> None:
+        response_guest_wifi = '''
+        {
+            "success": true,
+            "data": {
+                "guest_2g_enable": "on",
+                "guest_2g_ssid": "guest_2g",
+                "guest_2g_encryption": "psk",
+                "guest_2g_psk_key": "password",
+                "guest_2g_portal_enable": "off",
+                "guest_5g_enable": "off",
+                "guest_5g_ssid": "guest_5g",
+                "guest_5g_encryption": "none",
+                "guest_5g_psk_key": ""
+            }
+        }
+        '''
+
+        class TPLinkRouterTest(TplinkRouter):
+            def request(self, path: str, data: str,
+                        ignore_response: bool = False, ignore_errors: bool = False) -> dict | None:
+                if path == 'admin/wireless?form=guest_2g&form=guest_5g&form=guest_2g5g':
+                    return loads(response_guest_wifi)['data']
+                raise ClientException()
+
+        client = TPLinkRouterTest('', '')
+        guest_wifi_info = client.get_guest_wifi_info()
+
+        self.assertIsInstance(guest_wifi_info, GuestWifiStatus)
+        self.assertEqual(guest_wifi_info.guest_2g_enable, True)
+        self.assertEqual(guest_wifi_info.guest_2g_ssid, "guest_2g")
+        self.assertEqual(guest_wifi_info.guest_2g_encryption, "psk")
+        self.assertEqual(guest_wifi_info.guest_2g_psk_key, "password")
+        self.assertEqual(guest_wifi_info.guest_2g_portal_enable, False)
+        self.assertEqual(guest_wifi_info.guest_5g_enable, False)
+        self.assertEqual(guest_wifi_info.guest_5g_ssid, "guest_5g")
+        self.assertEqual(guest_wifi_info.guest_5g_encryption, "none")
+        self.assertEqual(guest_wifi_info.guest_5g_psk_key, "")
+
     def test_get_status(self) -> None:
```

Comment:
it is better to add to old test only checks that your new fields are None for case when router doesnt have these fields. And add a new test that checks all fields values with your new fields.